oak-pojosr/pom.xml

```diff
@@ -365,5 +365,11 @@
       <artifactId>testcontainers</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>testcontainers</artifactId>
+      <version>${testcontainers.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
same here

Reply:
removed in https://github.com/apache/jackrabbit-oak/pull/1459